targon/internal/targon/callbacks.go

```diff
@@ -396,16 +433,16 @@ func setWeights(v *boilerplate.BaseChainSubscriber, c *Core, h types.Header) {
 	c.Deps.Log.Infow("Set weights on chain successfully", "hash", hash.Hex())
 }
 
-func getWeights(c *Core) ([]types.U16, []types.U16, error) {
-	// TODO some sort of multi-check per interval
+func getWeights(c *Core) ([]types.U16, []types.U16, map[string]float64, error) {
```

Comment:
the signature of this function should not change. miner scores is useless for getting total alpha emission.